myosuite/tests/test_envs.py

```diff
@@ -22,9 +22,15 @@ def assert_close(prm1, prm2, atol=1e-05, rtol=1e-08):
         prm1_dict = flatten(prm1)
         prm2_dict = flatten(prm2)
         for key in prm1_dict.keys():
-            assert_close(prm1_dict[key], prm2_dict[key], atol=atol, rtol=rtol)
+            try:
+                assert_close(prm1_dict[key], prm2_dict[key], atol=atol, rtol=rtol)
+            except:
+                from pudb import set_trace; set_trace()
     else:
-        np.testing.assert_allclose(prm1, prm2, atol=atol, rtol=rtol)
+        try:
```

Comment:
This again seems like a debug statement. We need to catch these and tests needs to fail for these.

Reply:
fixed 91689968044638a33666ce48d07a99b86d781180